exampleSite/utilities/examples/content/events/yyyy-city/index.md

```diff
@@ -0,0 +1,86 @@
++++
+Title = "Welcome"
+Type = "welcome"
+aliases = ["/events/yyyy-city/welcome"]
++++
+
+<!-- <div style="text-align:center;">
+  {{< event_logo >}}
+</div> -->
+
+<div class = "row">
+  <div class = "col-md-2">
```

Comment:
I don't especially love recycling the same welcome page content as we did in the old defaults, but until I have something better this will be fine. 

Some of this stuff ends up being super redundant with the new CTA buttons, etc, but none of this is a blocker.